libs/rtemodel/src/RteGenerator.cpp

```diff
@@ -139,32 +200,33 @@ string RteGenerator::GetExpandedCommandLine(RteTarget* target) const
     if (p)
       cmd = p->GetAbsolutePackagePath() + cmd;
   }
+  return cmd;
+}
 
-  RteItem* argsItem = GetArgumentsItem("exe");
-  if (argsItem) {
+string RteGenerator::GetExpandedCommandLine(RteTarget* target) const
```

Comment:
Is there any change in the logic apart from using an intermediate structure?

Reply:
Not really. It just seemed to make sense to have a function to return arguments in a "refined" form.

Reply:
I have now removed `GetArguments()` (see discussion above).